### PR TITLE
updated the read me file for missing '/' for document templates config and fixed the openxml code for replacing the databindings correctly for fields and repeatable table rows.

### PR DESCRIPTION
**Pull request by ahmet269**
… the code for updating word templates before pushing them into target instance/s.

## Purpose
_Describe the problem or feature in addition to a link to the issue(s)._
the problem was the word templates with dataBinding tags were not deployed. the existing implementation tried to find and replace wpath for the binding in all document inner xml which ends up clearing the whole document and breaking the tag structure in the document.xml file for the template. 

## Approach
_How does this change address the problem?_

it now find all dataBinding descendant tags in the documentxml and it sets their prefixMapping property as expected. this allows the entitytypecodes to be set correctly without affecting the structure of document.xml at all. we can have 2 different types of dataBindings. one on added fields and one on repeatable tables which are 2 different tags. Both are now handled.

### CODE REVIEW

**Review by tdashworth — CHANGES_REQUESTED** [review]

File: src/Capgemini.PowerApps.PackageDeployerTemplate/Adapters/CrmServiceAdapter.cs [hunk 1]
```diff
@@ -140,25 +140,8 @@ public ExecuteMultipleResponse UpdateStateAndStatusForEntityInBatch(EntityCollec
         }
 
         /// <inheritdoc/>
-        public void ImportWordTemplate(string filePath)
+        public void ImportWordTemplate(FileInfo fileInfo, string entityLogicalName, OptionSetValue templateType, string filePath)
```
Comment (ewingjm) [inline src/Capgemini.PowerApps.PackageDeployerTemplate/Adapters/CrmServiceAdapter.cs:143]: Technically a breaking change to the method signature although I doubt it's being called other than internally at the moment. Generally the sort of change we would avoid making.

Comment (tdashworth) [reply src/Capgemini.PowerApps.PackageDeployerTemplate/Adapters/CrmServiceAdapter.cs:143]: Ah yes, very good point, @ewingjm. I'll watch out for this next time but agree it's something we can forgive this time. Do you think we should increase the major version accordingly?

**Review by tdashworth — APPROVED** [review]

File: src/Capgemini.PowerApps.PackageDeployerTemplate/Adapters/CrmServiceAdapter.cs (hunk 1 above)
Comment (ewingjm) [reply src/Capgemini.PowerApps.PackageDeployerTemplate/Adapters/CrmServiceAdapter.cs:143]: We're still v0.x so I think it's OK to make the occasional small breaking change. In this case it does look like the logic made sense to be moved elsewhere as having the `CrmServiceAdapter` was responsible for too much.

Comment (ewingjm) [reply src/Capgemini.PowerApps.PackageDeployerTemplate/Adapters/CrmServiceAdapter.cs:143]: I'd just let it increment the patch version as normal 👍🏻

Comment (ahmet269) [reply src/Capgemini.PowerApps.PackageDeployerTemplate/Adapters/CrmServiceAdapter.cs:143]: Reason that was made was to make sure the sole responsibility of the CRMServiceAdapter is to interact with the service layer and nothing else. before it was calling Utilities to strip entitynames and etc which as far as Service Adapter is concerned should be passed in as params.

**Comment by ewingjm** [comment]
@ahmet269 the pull request title will be the default commit message for the squash commit. We're using [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/#summary) so it's helpful if you can create the pull request with a name that conforms to the spec and a concise message within GitHub's 72 character limit. Not an issue if not as this can be overridden by whoever does the merge.

**Review by tdashworth — APPROVED** [review]

**Pull request merged**